nginx-app/templates/default/single-redirect.conf.erb

```diff
@@ -0,0 +1,8 @@
+server {
```

Comment:
Please rename the template to `redirect-single`, so all three redirection templates are next to each other